common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -1128,14 +1128,34 @@ def get_parent_location(self, locator, **kwargs):
 
         :param locator: BlockUsageLocator restricting search scope
         """
+        def has_path_to_root(xblock, course):
```

Comment:
@awaisdar001 , I think this method could be useful in a plethora of places. For example, it could be nice to use this to decide if a discussion module is in the course tree.